features/machine/machine.feature

```diff
@@ -68,3 +68,31 @@ Feature: Machine features testing
       | https://machine-api-operator.openshift-machine-api.svc:8443/metrics          | # @case_id OCP-25652
       | https://cluster-autoscaler-operator.openshift-machine-api.svc:9192/metrics   | # @case_id OCP-26111
       | https://machine-approver.openshift-cluster-machine-approver.svc:9192/metrics | # @case_id OCP-26102
+
+
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-27627
```

Comment:
The two lines are not aligned with the following lines. Please use the same indention.